website/content/en/blog/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2022-10-10
+linkTitle: "Kubeflow on AWS Blog"
+description: "Welcome to the **blog** section of Kubeflow on AWS!"
```

Comment:
suggest removing bold in all the places in this page, it does not look good. (Maybe its subjective)